src/resources/elements/textfield.html

```diff
@@ -6,7 +6,7 @@
         <img src="res/info.svg" alt="Info"/>
       </tooltip>
     </label>
-    <input type="text" id.bind="id" list.bind="listID & attr" value.bind="value" placeholder.bind="placeholder">
+    <input type.bind="format" id.bind="id" list.bind="listID & attr" value.bind="value" placeholder.bind="placeholder">
```

Comment:
IIRC that's an Aurelia keyword to tell Aurelia that we want to set the HTML attribute rather than the DOM (JS) field named `list`